sklearn/cluster/tests/test_bicluster.py

```diff
@@ -1,29 +1,24 @@
 """Testing for Spectral Biclustering methods"""
 
 import numpy as np
+from pytest import mark
```

Comment:
It the rest of the code so far, we are using package level import (`import pytest`) then calling `@pytest.mark.*` decorators. This chose might be debatable, but I think it would be better to be consistent and use `import pytest` here.